controllers/grid/queries/QueriesGridHandler.inc.php

```diff
@@ -654,28 +677,14 @@ public function updateQuery($args, $request)
                         '\'>' .
                         __('notification.unsubscribeNotifications') .
                         '</a>'
-                ]]);
-            }
+                ]);
 
-            $assocSubmission = Repo::submission()->get($query->getAssocId());
-
-            try {
-                event(new DiscussionMessageSent(
-                    $query,
-                    $request->getContext(),
-                    $assocSubmission,
-                    $formData->setRecipientIds($added)->setSenderId($request->getUser()->getId())
-                ));
-            } catch (Swift_TransportException $e) {
-                $notificationMgr->createTrivialNotification(
-                    $currentUser->getId(),
-                    PKPNotification::NOTIFICATION_TYPE_ERROR,
-                    ['contents' => __('email.compose.error')]
-                );
-                trigger_error($e->getMessage(), E_USER_WARNING);
+                $mailable->recipients([$user]);
+
+                Mail::send($mailable);
             }
-            return \PKP\db\DAO::getDataChangedEvent($query->getId());
```

Comment:
@NateWr, this introduced a bug with a page not being refreshed after `Query Form` is saved and `note` not being actually updated. I see that `DAO::getDataChangedEvent` method is deprecated, should it be replaced with a new pattern?

Reply:
You can continue to use it for grids.